app/src/main/java/com/hover/stax/presentation/welcome/WelcomeFragment.kt

```diff
@@ -27,7 +27,7 @@ class WelcomeFragment : Fragment() {
             id = R.id.welcomeFragment
             val buttonText = getString(R.string.explore_btn_text)
             setContent {
-                WelcomeScreen(buttonText, { onClickGetStarted() }, { onClickLogin() }, showExploreButton = false)
+                WelcomeScreen(buttonText, { onClickGetStarted() }, { onClickLogin() }, showExploreButton = true)
```

Comment:
We are bringing back the explore button, yes?

Reply:
yes